src/slack/config.go

```diff
@@ -132,6 +132,35 @@ var redeployDialog = dialog{
         "name" : "git_repo",
         "data_source": "external"
       },
+      {
```

Comment:
I would again emphasize to not add these fields to redeploy dialog. Redeploy should be a light weight operation, and I would suggest you to write a redeploy.sh which just does `docker-compose up -d --build`
We have for the time-being implemented it as a stop and then deploy, but docker provides an functionality to quickkly rebuild and up the service.